lib/src/components/navigation/tabs.dart

```diff
@@ -4,12 +4,14 @@ class Tabs extends StatelessWidget {
   final int index;
   final ValueChanged<int> onChanged;
   final List<Widget> tabs;
+  final EdgeInsets? padding;
```

Comment:
Please use EdgeInsetsGeometry so that it can support EdgeInsetsDirectional

Reply:
Noted, i will replace it